### PR TITLE
Section "Doc" from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by 1ucian0**
The section **Doc** in `CONTRIBUTING.md` says:

>Review the parts of the documentation regarding the new changes and
> update it if it\'s needed.

To me, this section says nothing (besides not having a good writing style). I suggest to remove it.

### CODE REVIEW

**Comment by ajavadia** [comment]
There should be something that reminds people "if you make a change, make sure you update the associated docstrings and/or parts of the [documentation](https://github.com/Qiskit/qiskit/tree/master/docs/terra) that corresponds to it"?

File: test/python/visualization/test_dag_drawer.py [hunk 1]
```diff
@@ -35,7 +36,7 @@ def setUp(self):
 
     def test_dag_drawer_no_graphviz(self):
         """Test dag draw with no graphviz."""
-        with unittest.mock.patch('nxpd.pydot.find_graphviz', return_value=None) as _:
+        with patch('nxpd.pydot.find_graphviz', return_value=None) as _:
```
Comment (ajavadia) [inline test/python/visualization/test_dag_drawer.py]: the changes to this file seem unrelated.

Comment (1ucian0) [reply test/python/visualization/test_dag_drawer.py]: solved in 35be0787

**Comment by 1ucian0** [comment]
Section back with your text. Also, some style issues.
